Tailscale/shared/Tailscale.sh

```diff
@@ -12,24 +15,31 @@ case "$1" in
         echo "${QPKG_NAME} is disabled."
         exit 1
     fi
-    mkdir -p -m 0755 /tmp/tailscale
-    if [ -e /tmp/tailscale/tailscaled.pid ]; then
-        PID=$(cat /tmp/tailscale/tailscaled.pid)
+
+    if [ -e ${PIDF} ]; then
+        PID=$(cat ${PIDF})
         if [ -d /proc/${PID}/ ]; then
           echo "${QPKG_NAME} is already running."
           exit 0
         fi
     fi
-    ${QPKG_ROOT}/tailscaled --port 41641 --statedir=${QPKG_ROOT}/state --socket=/tmp/tailscale/tailscaled.sock 2> /dev/null &
-    echo $! > /tmp/tailscale/tailscaled.pid
+    /bin/ln -sf ${QPKG_ROOT}/tailscaled /usr/bin/tailscaled
+    /bin/ln -sf ${QPKG_ROOT}/tailscale /usr/bin/tailscale
+    tailscaled --port 41641 --statedir=${QPKG_ROOT}/state 2> /dev/null &
+    echo $! > ${PIDF}
+    sleep 10
+    tailscale up 2>&1 > /dev/null | tee ${QPKG_ROOT}/tailscale.txt &
```

Comment:
The intent for the code on the main branch is to be run from the App Center, which forks `tailscale web` to authenticate. Running `tailscale up` here to print an AuthURL to a file is not the right thing to do.

If you want to operate the QNAP Tailscale interface primarily from the CLI you're welcome to,  but should do so via SSH to run `tailscale up` instead of a `${QPKG_ROOT}/tailscale.txt` file.

You're also welcome to use the branch which doesn't contain any of the `tailscale web` support: https://github.com/tailscale/tailscale-qpkg/tree/update

Once this package is in the App Center, most users won't ever use the CLI.

Reply:
When the Tailscale package is updated, you will need to connect again from the App Center. That was something I was trying to resolve, any other recommended approaches?

Running from the App Center is limited in what you can do, so my goal was to add the additional capabilities from the CLI.

Reply:
It is fine to use the CLI.

It is not fine to include `tailscale up` in the Tailscale.sh startup script. This PR will not be merged while that code is present.